src/runway/commands/runway/test.py

```diff
@@ -1,191 +1,50 @@
 """The test command."""
-
 import logging
-import os
 import sys
+import traceback
 
-from subprocess import check_call, check_output
-from subprocess import CalledProcessError
-
-# from stacker.util import parse_cloudformation_template
-# parse_cloudformation_template wraps yaml_parse; it would be better to call it
-# from util but that would require sys.path shenanigans here
-from runway.embedded.stacker.awscli_yamlhelper \
-    import yaml_parse as parse_cloudformation_template  # noqa
-from runway.util import (
-    change_dir, get_embedded_lib_path, ignore_exit_code_0, use_embedded_pkgs,
-    which
-)
-
-from ..runway_command import RunwayCommand
+from ..base_command import BaseCommand
+from ...tests.regestry import TEST_HANDLERS
```

Comment:
Let's rename this to `registry`

Reply:
fixed the type for the import and the filename...